lightningd/peer_control.c

```diff
@@ -2789,6 +2798,7 @@ static struct command_result *json_setchannel(struct command *cmd,
 		   p_opt("htlcmin", param_msat, &htlc_min),
 		   p_opt("htlcmax", param_msat, &htlc_max),
 		   p_opt_def("enforcedelay", param_number, &delaysecs, 600),
+		   p_opt("ignorefeelimits", param_bool, &ignore_fee_limits),
```

Comment:
Why do we not use the `ignore_fee_limits` as keyword? I feel that using this one-word style, we will make the autogen code a little bit ugly?

Reply:
It's consistent though :(

Reply:
yeah I know, now there is the was of pre-code gen and post code gen ahha